operator/grafana/generate-configmap.sh

```diff
@@ -0,0 +1,15 @@
+#! /bin/bash
```

Comment:
why not put the entire `operator/grafana` directory into the config directory? If its not included in the kustomization, it is ignored.

Reply:
I think config it's better just have resources yaml files, by the definition here https://book-v1.book.kubebuilder.io/basics/project_creation_and_structure.html
```
config/...

Kubebuilder creates yaml config for installing the CRDs and related objects under config/.
```
and this plugin also lack of the feature to config output (maybe another PR to support it😄)

Reply:
the generated configmap already in /config directory, which include the grafana dashboard content as `data:`, and they are under kustomization control.
the file in `operator/grafana` is just pure grafana dashboard json, it can be used for manually import.